tests/query/atomWithQuery.test.tsx

```diff
@@ -722,3 +730,47 @@ describe('error handling', () => {
     await findByText('count: 3')
   })
 })
+
+it('query expected QueryCache test', async () => {
+  const queryClient = new QueryClient()
+  const countAtom = atomWithQuery(
+    () => ({
+      queryKey: ['count6'],
+      queryFn: async () => {
+        return await fakeFetch({ count: 0 }, false, 100)
+      },
+    }),
+    () => queryClient
+  )
+  const Counter = () => {
+    const [
+      {
+        response: { count },
+      },
+    ] = useAtom(countAtom)
+    const cachedQueries = useSyncExternalStore(
```

Comment:
Please don't use uSES here.

I think you can just expect the result?
At L775, `expect(queryClient.getQueryCache().getAll().length).toBe(1)`.

Reply:
Good catch, done.